src/shared/scrapers/USA/GA/index.js

```diff
@@ -179,27 +179,64 @@ const scraper = {
     'Wilkinson County',
     'Worth County'
   ],
-  async scraper() {
-    const $ = await fetch.page(this.url);
-    let counties = [];
-    const $table = $('table:contains(County):contains(Cases)');
-    const $trs = $table.find('tbody > tr');
-    $trs.each((index, tr) => {
-      const $tr = $(tr);
-      const name = $tr.find('td:first-child').text();
-      let county = geography.addCounty(parse.string(name.replace('Dekalb', 'DeKalb')));
-      const cases = parse.number($tr.find('td:last-child').text());
-      if (county === 'Unknown County') {
-        county = UNASSIGNED;
-      }
-      counties.push({ county, cases });
-    });
+  _countyMap: {
+    Mcduffie: 'McDuffie',
+    Dekalb: 'DeKalb'
+  },
+  scraper: {
+    '0': async function() {
+      const $ = await fetch.page(this.url);
+      let counties = [];
+      const $trs = $('table:contains(County):contains(Cases) tbody > tr');
+      $trs.each((index, tr) => {
+        const $tr = $(tr);
+        let name = $tr.find('td:first-child').text();
+        name = this._countyMap[name] || name;
+        let county = geography.addCounty(parse.string(name));
+
+        const cases = parse.number($tr.find('td:nth-child(2)').text());
+
+        if (county === 'Unknown County') {
+          county = UNASSIGNED;
+        }
 
-    counties.push(transform.sumData(counties));
+        counties.push({ county, cases });
+      });
 
-    counties = geography.addEmptyRegions(counties, this._counties, 'county');
+      counties.push(transform.sumData(counties));
+      counties = geography.addEmptyRegions(counties, this._counties, 'county');
 
-    return counties;
+      return counties;
+    },
+    '2020-3-27': async function() {
+      const pageHTML = (await fetch.page(this.url)).html();
+      [this.url] = pageHTML.match(/https:\/\/(.*)\.cloudfront\.net/);
+
+      const $ = await fetch.page(this.url);
+      let counties = [];
+      const $trs = $('table:nth-child(6) tbody tr:not(:first-child,:last-child)');
```

Comment:
The way this table is being found is a little brittle -- is there an ID or a column with a name you can use?

```suggestion
      const $trs = $('td:contains("COVID-19 Confirmed Cases By County") tbody tr:not(:first-child,:last-child)').closest('table')
```

Reply:
This works 
```
$('.tcell:contains("COVID-19 Confirmed Cases By County")')
        .closest('tbody')
        .find('tr:not(:first-child,:last-child)')
```